dot/parachain/pvf/worker.go

```diff
@@ -0,0 +1,130 @@
+package pvf
+
+import (
+	"sync"
+
+	parachainruntime "github.com/ChainSafe/gossamer/dot/parachain/runtime"
+	parachaintypes "github.com/ChainSafe/gossamer/dot/parachain/types"
+)
+
+type worker struct {
+	workerID parachaintypes.ValidationCodeHash
+	instance *parachainruntime.Instance
+	queue    chan *workerTask
```

Comment:
This queue channel has capacity of only one. A new workerTask cannot be added until the existing one finishes. Make this a buffered channel.